### DISCUSSION TITLE
I can't connect to localhost

### DISCUSSION THREAD

**Discussion by RandomGames11** [mircokroon/minecraft-world-downloader#207] [Q&A]
everytime i try to local host i get this message. Cannot connect to (server ip). The server may be down or on a different address. (java.net.UnknownHostException). how can i fix this?
EDIT: i've tried this with multiple servers and both times this error message came up

**Comment by LilBamaBeets** [comment]
Run Minecraft.
On the main menu, click Multiplayer.
Click Add Server.
Name your server in the Server Name field.
Type "localhost" in the Server Address field.
Click Done.
Select your server and click Join Server

**Comment by WhynotFulp** [comment]
@LilBamaBeets @RandomGames11 Was this issue ever resolved? If so can someone share screen shots of how to connect and start the world download?
